Bottom: 866a67ebcac033f3f1d43b853a4ec0bc17df7000
Top:    45b48362ef0c09109b1cc50ae3db9ad2b142337e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 20:35:04 +0100

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 2637495..94c6779 100644
--- a/configure.ac
+++ b/configure.ac
@@ -393,8 +393,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
